models/prices/cardano/prices_cardano_tokens.sql

```diff
@@ -0,0 +1,20 @@
+{{ config(
+        schema='prices_cardano',
+        alias = 'tokens',
+        materialized='table',
+        file_format = 'delta',
+        tags = ['static']
+        )
+}}
+SELECT 
+    token_id
+    , blockchain
+    , symbol
+    , contract_address
```

Comment:
i believe you accidentally left this column in, which is the current CI failure. removing this should help proceed.